Unvendor sqlite3 from stdlib (#2946)
pyodide/pyodide: docs/project/changelog.md

  `append`, and `pop` for `JsProxy` of Javascript arrays.
  {pr}`2970`

> - {{ Breaking }} \_sqlite3 stdlib module is not unvendored by default.
>   Now it needs to be maually loaded by {any}`pyodide.loadPackage`
>   or {any}`micropip.install`.

Reviewer: ```suggestion
- {{ Breaking }} Unvendored the \_sqlite3 module from the standard library.
  Before `sqlite3` was included by default. Now it needs to be loaded with 
  {any}`pyodide.loadPackage` or {any}`micropip.install`.
```
